homeassistant/components/plugwise/__init__.py

```diff
@@ -94,6 +94,10 @@ async def async_update_data():
 
     api.get_all_devices()
 
+    if entry.unique_id is None:
+        if api.smile_version[0] != "1.8.0":
```

Comment:
Does this mean that some devices won't get a unique id?

Reply:
No, this means that for some devices the unique_id will not be updated.
But for these devices we have changed nothing so there should not be any issue.

Reply:
But these entries will still not have a unique id. I guess the same host check in the config flow should guard from setting up these again.

Reply:
Yes they were given a unique_id before and we are still giving these devices the same id with the updated code.